commerce_coordinator/apps/frontend_app_payment/urls.py

```diff
@@ -4,9 +4,10 @@
 
 from django.urls import path
 
-from commerce_coordinator.apps.frontend_app_payment.views import PaymentGetView
+from commerce_coordinator.apps.frontend_app_payment.views import DraftPaymentCreateView, PaymentGetView
 
 app_name = 'frontend_app_payment'
 urlpatterns = [
     path('payment/', PaymentGetView.as_view(), name='get_payment'),
+    path('payment/draft/', DraftPaymentCreateView.as_view(), name='creat_draft_payment'),
```

Comment:
updated spec URL from 
/frontend-app-payment/payment::PUT
to
/frontend-app-payment/payment/draft/::PUT

as we have an existing path for /frontend-app-payment/payment